backend/services/agent_create.py

```
@@ -9,6 +9,7 @@
 from dotenv import load_dotenv
 import os
 import httpx
+from app.core.auth import get_current_user
```

Ruff (0.8.2)

`app.core.auth.get_current_user` imported but unused

Remove unused import: `app.core.auth.get_current_user`

(F401)